src/main/java/uk/gov/hmcts/divorce/common/notification/AwaitingConditionalOrderNotification.java

```diff
@@ -32,16 +33,13 @@
 
 @Component
 @Slf4j
+@RequiredArgsConstructor
 public class AwaitingConditionalOrderNotification implements ApplicantNotification {
 
-    @Autowired
-    private CommonContent commonContent;
-
-    @Autowired
-    private NotificationService notificationService;
-
-    @Autowired
-    private ApplyForConditionalOrderPrinter applyForConditionalOrderPrinter;
```

Comment:
Looks like this printer class isn't being used any more? Can it be deleted?

Reply:
Yep - deleted